starter-code/src/functions-and-arrays.js

```diff
@@ -52,7 +156,27 @@ const wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist(arrayOfWords, anyOtherWord) {
+  let trueOrFalse = ""
+  for (let i = 0; i < wordsFind.length; i++) {
+    if (wordsFind[i] === anyOtherWord) {
+      //return true
+      trueOrFalse = "true"
+      break;
+    }
+    else {
+      //return false
+      trueOrFalse = "false"
+      continue;
+    }
+  }
+  return trueOrFalse
+}
+
+doesWordExist(wordsFind, "starting")
+
 // Iteration #7: Count repetition
+
```

Comment:
awesome job, i really like your solutions  if i wanna be picky i could say its better to your functions has "return" instead of console.log() then when you wanna check the result you could console.log the function itself